docs/modules/nonlinearities.rst

```diff
@@ -3,5 +3,20 @@
 
 .. automodule:: lasagne.nonlinearities
 
+.. autosummary::
+
+   sigmoid
```

Comment:
I don't know any way to avoid this without wrapping it in its own function.

However, it seems not too bad to do so. It is very easy to read and only one line of "overhead".

I could also adjust the docstring in Theano. That would probably be the best solution. However, I am not too sure what to write for softmax there.